config/initializers/foreman.rb

```diff
@@ -6,14 +6,14 @@
 require_dependency 'foreman/access_permissions'
 require_dependency 'foreman/plugin'
 require_dependency 'foreman/settings'
+require_dependency 'fog_extensions'
 
 # Other internal dependencies, may be autoloaded
-require_dependency 'foreman/foreman_url_renderer'
-require_dependency 'foreman/controller'
-require_dependency 'net'
-require_dependency 'foreman/provision' if SETTINGS[:unattended]
-require_dependency 'foreman'
-require_dependency 'fog_extensions'
+require 'foreman/foreman_url_renderer'
+require 'foreman/controller'
+require 'net'
+require 'foreman/provision' if SETTINGS[:unattended]
```

Comment:
hehe this is even funnier case. It's doing exactly what Zeitwerk will do with a proper inflector setup of (SSH => ssh).
It probably won't work now without it, but we should have a followup task for that.

Reply:
https://projects.theforeman.org/issues/29991